### DISCUSSION TITLE
Support for paper_trail

### DISCUSSION THREAD

**Discussion by jmbds** [avo-hq/avo#822] [Help]
I have versioning in my models using [paper_trail gem](https://github.com/paper-trail-gem/paper_trail) and I'd like to display the versions in the show view of each of my resources.

Is there a way to do this? Creating a version resource and adding a has_many :versions field to my versioned resources doesn't seem to work.

Thanks!

**Comment by adrianthedev** [comment]
You probably need a custom field for that. I'm not very familiar with `paper_trail` but I'd love to jump on a call to know more about how you need to have that integrated.

Do you mind [booking a slot](https://calendly.com/adrianthedev) at your earliest convenience?
